src/Calendar.js

```diff
@@ -46,8 +49,16 @@ export default class Calendar extends Component {
     const daysInMonth = getDaysInMonth(this.state.date) + 1
 
     for (let dateIndex = 1; dateIndex < daysInMonth; dateIndex += 1) {
-      const isDisabled = isBefore(new Date(format(this.state.date, `MM/${dateIndex}/YYYY`)), new Date(format(this.props.minDate, 'MM/DD/YYYY')))
-
+      const isDisabled =
+        isBefore(
+          new Date(format(this.state.date, `MM/${dateIndex}/YYYY`)),
```

Comment:
I don't think we need to create the same date object multiple times. How about a const?
```
const dt = new Date(format(this.state.date, `MM/${dateIndex}/YYYY`)),

const isDisabled =
        isBefore(dt, new Date(format(this.props.minDate, 'MM/DD/YYYY'))) 
        ||
        isAfter(dt, new Date(format(this.props.maxDate, 'MM/DD/YYYY')))
``

Reply:
Also, will this work if `minDate` or `maxDate` isn't defined?

Reply:
I'll update the const, but yeah, isAfter handles null cases, so it still works. I tested on the UI as well.